rules-reviewed/eap7/eap4and5/jboss-eap4and5to6and7-xml.windup.xml

```diff
@@ -71,7 +70,7 @@
                 <hint issue-display-mode="all" title="Replace CompressingMarshaller marshaller" effort="1" category-id="mandatory">
                     <message>
                         <![CDATA[
-                        `CompressingMarshaller` marshaller is no longer available in JBoss EAP 6+.  
```

Comment:
is this double space removal on purpose ? we put it to create a new line after the text.